include/pagmo/problems/wfg.hpp

```diff
@@ -0,0 +1,138 @@
+/* Copyright 2017-2018 PaGMO development team
+
+This file is part of the PaGMO library.
+
+The PaGMO library is free software; you can redistribute it and/or modify
+it under the terms of either:
+
+  * the GNU Lesser General Public License as published by the Free
+    Software Foundation; either version 3 of the License, or (at your
+    option) any later version.
+
+or
+
+  * the GNU General Public License as published by the Free Software
+    Foundation; either version 3 of the License, or (at your option) any
+    later version.
+
+or both in parallel, as here.
+
+The PaGMO library is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received copies of the GNU General Public License and the
+GNU Lesser General Public License along with the PaGMO library.  If not,
+see https://www.gnu.org/licenses/. */
+
+#ifndef PAGMO_PROBLEMS_WFG_HPP
+#define PAGMO_PROBLEMS_WFG_HPP
+
+#include <string>
+#include <utility>
+
+#include <pagmo/detail/visibility.hpp>
+#include <pagmo/population.hpp>
+#include <pagmo/problem.hpp>
+#include <pagmo/types.hpp>
+
+namespace pagmo
+{
+
+/// WFG problem test suite
+/**
```

Comment:
Could you please remove all doxygen-style docs from the C++ source files?

Reply:
Shall I remove them all (and leave just the very basic descriptions), or just turn them into normal comments?

Reply:
Leave basic one-line descriptions and don't use any special doxygen syntax (meaning, don't use triple slashes ``///`` or the slash-star patterns ``/**``).